server/paymentProviders/opencollective/host.js

```diff
@@ -69,9 +70,9 @@ paymentMethodProvider.processOrder = async order => {
     },
   };
 
-  const transactions = await models.Transaction.createFromPayload(payload);
-
-  return transactions;
+  if (payload.transaction.amount > 0) {
+    return await models.Transaction.createFromPayload(payload);
+  }
```

Comment:
Sounds like we don't need this anymore?

```suggestion
  return models.Transaction.createFromPayload(payload);
```